cinema_service/settings.py

```diff
@@ -86,12 +87,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB", "cinema"),
+        "USER": os.environ.get("POSTGRES_USER", "cinema"),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", "cinema"),
+        "HOST": os.environ.get("POSTGRES_HOST", "db"),
```

Comment:
Ensure that there is a mechanism, such as a `wait_for_db` command, to wait for the database to be ready before the application starts. This is crucial to prevent connection errors at startup.